crates/wasmtime/src/func.rs

```diff
@@ -578,22 +599,32 @@ impl Func {
         &self.export
     }
 
+    pub(crate) fn caller_checked_anyfunc(
+        &self,
+    ) -> NonNull<wasmtime_runtime::VMCallerCheckedAnyfunc> {
+        self.export.anyfunc
+    }
+
     pub(crate) fn from_wasmtime_function(
         export: wasmtime_runtime::ExportFunction,
         instance: StoreInstanceHandle,
-    ) -> Self {
+    ) -> Option<Self> {
+        if unsafe { export.anyfunc.as_ref().func_ptr.is_null() } {
```

Comment:
I think this answers my question about where the `None` came from, but I think that we're guaranteed `ExportFunction` is never a "null function", right?